src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -121,7 +122,7 @@ class WorkspacePageWithSections extends React.Component {
                     />
                     {this.props.shouldUseScrollView
                         ? (
-                            <ScrollView
+                            <ScrollViewWithPickers
```

Comment:
Is this the only place right now where we have a scroll view with pickers, or does this need to be added to any other places?

Reply:
There's a Picker in "General Settings" under Workspace settings, which is also covered by the change in `src/pages/workspace/WorkspacePageWithSections.js`.

Added it for ProfileScreen (in `src/components/Forms.js`) as well